From 77b5ea1f3d75214b68513f327dabebaebafe9d8e Mon Sep 17 00:00:00 2001
From: EdenEast <edenofest@gmail.com>
Date: Thu, 26 Oct 2023 19:43:19 -0400
Subject: [PATCH] patch version

---
 bin/asciinema-rec_script | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/bin/asciinema-rec_script b/bin/asciinema-rec_script
index 4bbeb95..83aa1be 100755
--- a/bin/asciinema-rec_script
+++ b/bin/asciinema-rec_script
@@ -16,8 +16,7 @@ export COMMENT="${COMMENT:-"💭 [3;1m"}"
 export ECHO="${ECHO:-"💬 [3;1m"}"
 export BEGIN_RECORDING="${BEGIN_RECORDING:-"🎥 [32;3mBegin recording - [1m"}"
 export END_RECORDING="${END_RECORDING:-"🎬 [32;3mEnd recording - [1m"}"
-export VERSION="${VERSION:-$(cat "${BASH_SOURCE[0]%/*}/../version.txt")}"
-
+export VERSION="v0.10.0"
 
 build_augmented_script() { local script="$1"
   comment() {
-- 
2.41.0

